pages/audit.php

```diff
@@ -139,27 +140,15 @@ function GetRuleResultFilter($iRuleId, $oDefinitionFilter, $oAppContext)
 	return $oFilter;
 }
 
-function GetReportColor($iTotal, $iErrors)
-{
-	$sResult = 'red';
-	if ( ($iTotal == 0) || ($iErrors / $iTotal) <= 0.05 )
-	{
-		$sResult = 'green';
-	}
-	else if ( ($iErrors / $iTotal) <= 0.25 )
-	{
-		$sResult = 'orange';
-	}
-	return $sResult;
-}
-
 try
 {
 	require_once('../approot.inc.php');
 	require_once(APPROOT.'/application/application.inc.php');
 
 	require_once(APPROOT.'/application/startup.inc.php');
-	$operation = utils::ReadParam('operation', '');
+
+	$bSelectionAuditRulesByDefault = MetaModel::GetConfig()->Get('audit.enable_selection_landing_page');
```

Comment:
```suggestion
	$bSelectionAuditRulesByDefault = utils::GetConfig()->Get('audit.enable_selection_landing_page');
```

We tend to always retrieve conf params through the `utils` class now as when doing so through the `MetaModel` class it can fail if the MetaModel has not been started yet.
In this particular case it will always be started, but as people tend to copy/paste pattern we rather keep the good one as much as possible 😁

Reply:
Check :-)
This is done